blurring_as_a_service/cleaning_pipeline/submit_cleaning_pipeline.py

```diff
@@ -33,13 +33,25 @@ def cleaning_pipeline():
         database_parameters_json=database_parameters_json,
         customer_name=customer_name,
     )
-    customer_in_cvt_path = aml_interface.get_datastore_full_path(
-        f"{customer_name}_in_cvt"
+
+    customer_quality_check_path = aml_interface.get_datastore_full_path(
+        f"{customer_name}_quality_check"
+    )
+
+    smart_sampling_step.outputs.customer_quality_check_folder = Output(
+        type=AssetTypes.URI_FOLDER,
+        mode="rw_mount",
+        path=customer_quality_check_path,
+    )
+
+    customer_retraining_path = aml_interface.get_datastore_full_path(
+        f"{customer_name}_retraining"
     )
-    smart_sampling_step.outputs.customer_cvt_folder = Output(
+
+    smart_sampling_step.outputs.customer_retraining_folder = Output(
```

Comment:
can you please add a description for the Inputs and Outputs objects? also the ones that are commented from the deletion phase,

like this:
```
 smart_sampling_step.outputs.customer_quality_check_folder = Output(
         type=AssetTypes.URI_FOLDER,
         mode="rw_mount",
         path=customer_quality_check_path,
         description="a short description"
```